pkg/catalog/catalog.go

```diff
@@ -29,5 +33,17 @@ func (c *Catalog) Search(name string) []profilesv1.ProfileDescription {
 			profiles = append(profiles, p)
 		}
 	}
+
```

Comment:
did you want fix the `strings.Contains` to fail if `name` is empty string? Unrelated to the PR, but spotted during testing get :wink:

Reply:
should we fail or just return nothing?

Reply:
meh will do it later